src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml

```diff
@@ -30,7 +30,8 @@
         <StackPanel Style="{StaticResource SettingsStackStyle}">
             <!--  Language  -->
             <local:SettingContainer x:Uid="Globals_Language"
-                                    Margin="0">
+                                    Margin="0"
+                                    Visibility="{x:Bind LanguageSelectorAvailable}">

```

Comment:
I was told to bind x:Load so it won't even try to load any of this stuff or its children versus Visibility (which loads the stuff and then just doesn't show it.) 

You might also consider a one-time binding since this isn't going to change over the lifetime. Might be less code generated.

Reply:
The default binding mode of `x:Bind` is actually already `OneTime`!
I didn't know about `x:Load` - I'll use that instead. 🙂

Reply:
default for x:Bind is OneTime fortunately :smile: